src/constants.ts

```diff
@@ -38,3 +38,29 @@ export const TRUSTED_PROXIES = [
       .join('')
   ),
 ];
+export const API = {
+  FILTER: 'filter',
+  RISK: 'risk'
+};
+export const EVENT_TYPE = {
```

Comment:
If we could create a separate file with the EventType enum model and assign it to the Payload typings